.buildkite/cd/_forge.rayci.yml

```diff
@@ -0,0 +1,60 @@
+group: forge
```

Comment:
are these always going to be the same? you can symlink the file if they are.

Reply:
good idea!